index.html

```diff
@@ -1808,77 +1894,252 @@ <h2>Authentication</h2>
+is during the establishment of a TLS session on behalf of the <a>DID
+Subject</a>. In this case, the counterparty checks that the <a>verification
+method</a> used during the protocol handshake is associated with the
+<code>keyAgreement</code> property in the <a>DID Document</a>.
       </p>
 
       <p>
-A DID document MAY include a <code>controller</code> property to indicate
-that there are <a>DID controller(s)</a> other than the <a>DID subject</a>.
-If so:
+A <a>DID document</a> MAY include an <code>keyAgreement</code> property.
+The <code>keyAgreement</code> property is an additional example of a
+<a>verification relationship</a>.
       </p>
 
-      <dl>
-          <dt><dfn>controller</dfn></dt>
-          <dd>
-The value of the <code>controller</code> property MUST be a valid <a>DID</a>
-or an array of valid <a>DIDs</a>. The corresponding <a>DID document</a>(s)
-SHOULD contain authorization relations that explicitly permit the use of
-certain verification methods for specific purposes.
-          </dd>
-      </dl>
+      <p>
+Example:
+      </p>
 
+      <pre class="example nohighlight" title="Key agreement property
+      containing three verification methods">
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  "id": "did:example:123456789abcdefghi",
+  <span class="comment">...</span>
+  "keyAgreement": [
+    <span class="comment">// this method can be used to perform key agreement
+    as did:...fghi</span>
+    "did:example:123456789abcdefghi#keys-1",
+    <span class="comment">// this method is *only* authorized for key 
+      agreement usage, it may not</span>
+    <span class="comment">// be used for any other verification relationship, 
+      so its full description is</span>
+    <span class="comment">// embedded here rather than using only a 
+      reference</span>
+    {
+      "id": "did:example:123#zC9ByQ8aJs8vrNXyDhPHHNNMSHPcaSgNpjjsBYpMMjsTdS",
+      "type": "X25519KeyAgreementKey2019",
+      "controller": "did:example:123",
+      "publicKeyBase58": "9hFgmPVfmBZwRvFEyniQDBkz9LmV7gDEqytWyGZLmDXE"
+    }
+  ],
+}
+  <span class="comment">...</span>
+      
+      </pre>
+    </section>
+      
+    <section>
+      <h3>capabilityInvocation</h3>
       <p>
-Each <a>DID method</a> MUST define how authorization and delegation are
-implemented, including any necessary cryptographic operations.
+The <code>capabilityInvocation<code> property is used to express a
+<a>verification relationship</a> which an entity can use to invoke
+capabilities as the <a>DID Subject</a> or on behalf of the <a>DID
+subject</a>. A capability is an expression of an action that the <a>DID
+subject</a> is authorized to take. The <em>verifier</em> of a
+<code>capabilityInvocation<code> attempt can check proof of an invocation of a
+capability on behalf of the <a>DID subject</a> by checking if the
+<a>verification method</a> used with the proof is contained in the
+<code>capabilityInvocation<code> property of the <a>DID Document</a>.
+      </p>
```

Comment:
While the above paragraph is accurate, it's very difficult to follow. Try to simplify. If you don't get around to it, the Editors will try to clean it up in an editorial pass.

Reply:
I gave it an attempt, but probably still haven't quite hit the mark yet. Would be good to do the last fix ups to it with an editorial pass from this PR.